Examples/Framework/include/ACTFW/Utilities/Options.hpp

```diff
@@ -32,6 +32,8 @@ struct Interval {
   std::optional<double> upper;
 };
 
+using Intervals = std::vector<Interval>;
```

Comment:
I do not think it is a good idea to have this typedef. It hides information. If I read `value<std::vector<Interval>>` in an option description, I can see directly that it is an option that allows multiple values. If I only have `value<Intervals>` it could also be a different type that corresponds to a single input. With the typedef, I have to look it up to understand it. Without the typedef, I do not have to.

Reply:
You have a point, I can change that, was just lazy.

Reply:
Fixed.